notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            tf.keras.callbacks.LambdaCallback(on_epoch_end=model.plot_images),
</pre><p>I like this callback!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>